Project2015To2017/Facility.cs

```diff
@@ -0,0 +1,211 @@
+using Microsoft.Extensions.Logging;
+using Project2015To2017.Analysis;
+using Project2015To2017.Definition;
+using Project2015To2017.Migrate2017;
+using Project2015To2017.Reading;
+using Project2015To2017.Transforms;
+using Project2015To2017.Writing;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+
+namespace Project2015To2017
+{
+	public class Facility
```

Comment:
This is moving it down into the shared nupkg so it can be reused there, right?

Reply:
Yes, that's right.